airflow/models.py

```diff
@@ -1574,6 +1574,8 @@ def dry_run(self):
     def handle_failure(self, error, test_mode=False, context=None, session=None):
         self.log.exception(error)
         task = self.task
+        session = settings.Session()
```

Comment:
What is this line for? It seems like it does nothing?

Reply:
Cruft from testing, I suppose; I'll remove